Task2/TrainingManagementSystem/Training.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace TrainingManagementSystem
+{
+    internal class Training : TrainingComponent
```

Comment:
1. _trainingsAndExercises (code style for private fields)
2. List<T> is not allowed for this task, it was later on lectures